### DISCUSSION TITLE
Removing tags from refs page removes everything

### DISCUSSION THREAD

**Discussion by thegeorgeous** [magit/magit#4424] [Support] [answered]
Hiding the list of tags in the show-refs page as described [here](https://magit.vc/manual/magit/Performance.html#Refs-Buffer-Performance) hides all the sections and only shows the heading `HEAD`. Is this still the correct way to hide tags in the `show-refs` page? Or is the documentation outdated?

```
(remove-hook 'magit-refs-sections-hook 'magit-insert-tags)
```

**Comment by tarsius** [comment, marked as answer]
For me this still works.
